cli/remote_build.go

```diff
@@ -389,6 +389,10 @@ func fetchProject(urls []string) (*Project, error) {
 	if err != nil {
 		return nil, err
 	}
+	// TODO: Use the correct URL rather than the first one.
```

Comment:
Have the API return the original remote(s) URL(s) sent at the beginning, for yb to clone it locally as the user already does.